.gitlab-ci.yml

```diff
@@ -648,8 +648,8 @@ test_accep_vexpress_qemu_flash:
   after_script:
     - export WORKSPACE=$(realpath ${CI_PROJECT_DIR}/..)
     - if [ "$TEST_VEXPRESS_QEMU_FLASH" = "true" ]; then
-        cp ${CI_PROJECT_DIR}/../meta-mender/tests/acceptance/results.xml results_accep_vexpress_qemu_flash.xml;
-        cp ${CI_PROJECT_DIR}/../meta-mender/tests/acceptance/report.html report_accep_vexpress_qemu_flash.html;
+        cp $WORKSPACE/meta-mender/tests/acceptance/results.xml results_accep_vexpress_qemu_flash.xml;
+        cp $WORKSPACE/meta-mender/tests/acceptance/report.html report_accep_vexpress_qemu_flash.html;
       fi
```

Comment:
Why is this platform missing the artifact?

Reply:
It is commented above: https://github.com/mendersoftware/mender-qa/pull/341/files#diff-96edf7a6f008de9e928d04e1ae5e12a5R634-R636
```
# Note that vexpress-qemu-flash yoctobuild does not generate a ext4 image so
# the build script does not generate the Mender Artifact neither.
# Therefore this job has no deliverables to be passed to the release stage.
```

Reply:
Ah, but it could generate an artifact with the ubifs file. But anyway, it's not important, just wondered if it was missed by accident.

Reply:
Ok, thanks. That artifact was not being uploaded anywhere and that is why I removed it completely. But I think you are right and we must collect it here (so that we could for instance download it for manual testing).

I will follow-up on this at some point....